src/gui/log/loglistview.cpp

```diff
@@ -70,13 +70,16 @@ namespace
             painter->save();
             QStyledItemDelegate::paint(painter, option, index); // paints background, focus rect and selection rect
 
-            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();;
+            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();
             const QRect textRect = style->subElementRect(QStyle::SE_ItemViewItemText, &option, option.widget)
                                    .adjusted(1, 0, 0, 0); // shift 1 to avoid text being too close to focus rect
 
+            // for unknown reasons (fixme) painter won't accept some font properties
+            // until they are set explicitly, and we have to manually set some font properties
             QFont font = option.font;
```

Comment:
I don't really understand why copy constructor produces the object that behaves differently than the original one... 🤔
Did you try just `painter->setFont(option.font);`?

Reply:
with scaling 125%, 
![image](https://user-images.githubusercontent.com/34717789/83360163-d317cf00-a39c-11ea-9b7a-c281ce4ac846.png)

I tried printing their values both (option.font and font) are actually the same but somehow painter will not use these properties unless they are set explicitly.